### ISSUE TITLE
separate deadzones etc

### ISSUE THREAD

**Issue by rampancy777** [ericwa/Quakespasm#13]
it would be great if we could have separate deadzones for each axis vs one shared by both sticks.

also, would be cool if the movement stick had option of being treated as digital, meaning youre either walking or not (unless you press the run button). would be more like how the game plays with kb/m. 

might be easier to just say that run is not enabled on the joystick unless either the run button is depressed or the always run is enabled. 

basically i just want to be able to walk like normal without having to hold the thumbstick at the exact angle for walking speed. frantic gameplay is not conducive to this. 

thank you for the awesomeness
